# Message for a single detected object's bounding box
string object_class
float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
